web/templates/presence_row.hbs

```diff
@@ -4,11 +4,17 @@
             <span class="{{user_circle_class}} user_circle"></span>
             <a class="user-presence-link" href="{{href}}">
                 <div class="user-name-and-status-wrapper">
+                    {{#if status_text}}
+                    <div class="user-name-and-status-emoji">
+                        {{> user_full_name .}}
+                    </div>
+                    <span class="status-text"><span class="status-emoji">{{> status_emoji status_emoji_info}}</span> {{status_text}}</span>
```

Comment:
This would be easier to read like this:

```
<span class="status-text">
    <span class="status-emoji">{{> status_emoji status_emoji_info}}</span>
    {{status_text}}
</span>
```